docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,739 @@
+  </div>
+</div>
+
+Flutter is Google's user interface toolkit for building applications for mobile, web, and desktop from a single codebase.
+If you're looking to monitor and control individual machines with the same functionality you have on the [**Control** tab](/fleet/machines/#control), you can use the general-purpose [Viam mobile app](/fleet/#the-viam-mobile-app) rather than creating your own.
+If you need custom functionality or a custom interface, you can use Viam's [Flutter SDK](https://flutter.viam.dev/) to build a custom app to interact with your machines that run on Viam.
+
+This tutorial guides you through creating a mobile app that shows your machines and their components.
+As you work through this project you will learn the following:
+
+- The basics of how Flutter works
+- How to create _layouts_ in Flutter
+- How to connect user interactions (such as button presses) to app behavior
+- The basics of using Viam's Flutter SDK
+
+## Requirements
+
+You do not need any hardware for this tutorial other than a computer running macOS or a 64-bit Linux operating system.
+
+This tutorial assumes you already have a machine [configured](/build/configure/) on the [Viam app](https://app.viam.com).
+
+## Set up your Flutter environment
+
+This tutorial uses [Visual Studio Code](https://code.visualstudio.com/download) (VS Code) as the development environment (IDE), and uses the VS Code [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) to generate necessary boilerplate code.
+It's free, works on all major platforms, and has great integrations with Flutter.
+You can use a different editor, but it will be easier to follow along using VS Code.
+
+### Choose a development target
+
+Flutter can compile and run on many different platforms.
+For the purpose of this tutorial, we will be developing for iOS.
+
+You can always run your app on another platform later by developing with a different target.
+
+### Install Flutter
+
+Install Flutter according to [the Flutter documentation](https://docs.flutter.dev/get-started/install).
+The instructions on the Flutter website cover not only the installation of the SDK itself, but also the development target-related tools and the editor plugins.
+For this walkthrough, you only need to install the following:
+
+- Flutter SDK
+- Visual Studio Code with the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter)
+- The software required by your chosen development target.
+  For example:
+  - [Xcode](https://developer.apple.com/xcode/) to target macOS
+    - When prompted, do install Cocoapods.
+      You need it to support the iOS simulator.
+  - [Visual Studio](https://visualstudio.microsoft.com/) to target Windows
```

Comment:
```suggestion
- The software required by your chosen development target.
Since the development target in this tutorial is iOS, that is [Xcode](https://developer.apple.com/xcode/)
    - When prompted, do install Cocoapods.
      You need it to support the iOS simulator.
```
@nataliajacobowitz I had left Visual Studio in to help users who are referencing but diverging from this tutorial; do you think I should change it to this per conversation above?

Reply:
If it is not possible at all to do this tutorial with ease on windows, then sure lets remove. Otherwise why not leave that option for people? @clintpurser since you might be familiar with the process of doing this on windows - is it easy enough to make this whole thing work there or don't bother?

Reply:
we'd need to suggest they use some additional tooling which according to Reddit isn't ideal for devleopment. I think let's get this out the door as is for now and we can update the tutorial later